docs/installation/source.rst

```diff
@@ -37,36 +37,33 @@ please follow the directions :ref:`here <contributing>`.
        On Fedora Linux, you must replace ``pip`` with ``pip-python`` in the
        following steps.
 
-#. Then you'll need to install GStreamer 0.10 (>= 0.10.31, < 0.11), with Python
-   bindings. GStreamer is packaged for most popular Linux distributions. Search
-   for GStreamer in your package manager, and make sure to install the Python
+#. Then you'll need to install GStreamer 1.x (>= 1.2), with Python bindings.
+   GStreamer is packaged for most popular Linux distributions. Search for
+   GStreamer in your package manager, and make sure to install the Python
    bindings, and the "good" and "ugly" plugin sets.
 
    If you use Debian/Ubuntu you can install GStreamer like this::
 
-       sudo apt-get install python-gst0.10 gstreamer0.10-plugins-good \
-           gstreamer0.10-plugins-ugly gstreamer0.10-tools
+       sudo apt-get install python-gst-1.0 gstreamer1.0-plugins-good \
+           gstreamer1.0-plugins-ugly gstreamer1.0-tools
 
    If you use Arch Linux, install the following packages from the official
    repository::
 
-       sudo pacman -S gstreamer0.10-python gstreamer0.10-good-plugins \
-           gstreamer0.10-ugly-plugins
+       sudo pacman -S gst-python gst-plugins-good gst-plugins-ugly
```

Comment:
`gst-python` depends on `python-gobject`, while mopidy requires `python2-gobject`, since it's Python 2. I didn't find any `gst-python` for Python 2, but it works for me after installing `python2-gobject`. Not sure if `gst-python` is required. Probably not, since it's for Python 3, and it seems to work for me without it. It depends on `gst-plugins-base` though, which may be required(?)